sites/public/src/components/listings/ListingsList.tsx

```diff
@@ -48,7 +48,9 @@ const ListingsList = (props: ListingsListProps) => {
           subtitle={t("t.emergencyShelter")}
           className="is-normal-secondary-lighter"
         >
-          <Button className="is-secondary">{t("t.helpCenter")}</Button>
+          <LinkButton href="help/housing-help" className="is-secondary">
```

Comment:
I think this should start with / so /help/housing-help

Reply:
Geesh. How did I miss that. Tho it did work when I tested it locally. Fixed now! Thanks!